src/main.cpp

```diff
@@ -204,20 +204,24 @@ void publishMsgs(um6::Registers& r, ros::NodeHandle* imu_nh, sensor_msgs::Imu& i
     imu_msg.orientation_covariance[7] = r.covariance.get_scaled(14);
     imu_msg.orientation_covariance[8] = r.covariance.get_scaled(15);
 
-    // NED -> ENU conversion (x = y, y = x, z = -z)
+    // body-fixed frame NED to ENU: (x y z)->(x -y -z) or (w x y z)->(x -y -z w)
+    // world frame      NED to ENU: (x y z)->(y  x -z) or (w x y z)->(y  x -z w)
     if (tf_ned_to_enu)
     {
-      imu_msg.orientation.x = r.quat.get_scaled(2);
-      imu_msg.orientation.y = r.quat.get_scaled(1);
-      imu_msg.orientation.z = -r.quat.get_scaled(3);
-      imu_msg.orientation.w = r.quat.get_scaled(0);
-
-      imu_msg.angular_velocity.x = r.gyro.get_scaled(1);
-      imu_msg.angular_velocity.y = r.gyro.get_scaled(0);
+      // world frame
+      imu_msg.orientation.w =  r.quat.get_scaled(2);
+      imu_msg.orientation.x =  r.quat.get_scaled(1);
+      imu_msg.orientation.y = -r.quat.get_scaled(3);
+      imu_msg.orientation.z =  r.quat.get_scaled(0);
+
+      // body-fixed frame
+      imu_msg.angular_velocity.x =  r.gyro.get_scaled(0);
+      imu_msg.angular_velocity.y = -r.gyro.get_scaled(1);
       imu_msg.angular_velocity.z = -r.gyro.get_scaled(2);
 
-      imu_msg.linear_acceleration.x = r.accel.get_scaled(1);
-      imu_msg.linear_acceleration.y = r.accel.get_scaled(0);
+      // body-fixed frame
+      imu_msg.linear_acceleration.x =  r.accel.get_scaled(0);
+      imu_msg.linear_acceleration.y = -r.accel.get_scaled(1);
```

Comment:
If you're changing the behaviour of this package, you'll have to put the new behaviour behind a default-off flag to avoid breaking every other Husky out there.